### DISCUSSION TITLE
Statusline should be like in light-mod when it's Dark?

### DISCUSSION THREAD

**Discussion by Hohot** [ellisonleao/gruvbox.nvim#180] [Q&A]
Hello. I have a question about the statusline, is it should be soft in the dark-mod?
![image](https://user-images.githubusercontent.com/5588297/201991247-227e2a59-d074-496e-a63e-a4c818164623.png)

**Comment by Rafael-Conde** [comment]
Even if that's how it's intended to be, I would like to change it, since I find it better in the dark color, do you know how to change it? I've been searching for a while now for nothing so far.

**Comment by ellisonleao** [comment]
@Hohot which statusline plugin are you using? can you share some configs please?
